ArcGISRuntimeSDKQt_CppSamples/Geometry/FormatCoordinates/FormatCoordinates.cpp

```diff
@@ -177,3 +144,34 @@ QString FormatCoordinates::strUtm() const
 {
   return tr("Utm");
 }
+
+void FormatCoordinates::setMapView(MapQuickView* mapView)
+{
+  // find QML MapView component
+  m_mapView = mapView;
+
+  // create a new basemap instance
```

Comment:
- Would it be cleaner to still do much of this work in the constructor? And keep this setter specific to setting the MapView and possibly setting the map on the mapview, graphics overlay on the mapview, etc? 
- Could connectSignals be called from the main constructor instead of here?